File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/ForeignCompanyDetails.java
package com.babestudios.companyinfouk.data.model.company;

import com.google.gson.annotations.SerializedName;

public class ForeignCompanyDetails {

	@SerializedName("accounting_requirement")
	public AccountingRequirement accountingRequirement;

	@SerializedName("accounts")
	public ForeignCompanyAccounts accounts;

	@SerializedName("business_activity")
	public String businessActivity;

	@SerializedName("company_type")
	public String companyType;

	@SerializedName("governed_by")
	public String governedBy;

	@SerializedName("is_a_credit_finance_institution")
	public String isACreditFinanceInstitution;

	@SerializedName("originating_registry")
	public OriginatingRegistry originatingRegistry;

	@SerializedName("registration_number")
	public String registrationNumber;

}

File: data/src/main/java/com/babestudios/companyinfouk/data/model/company/ForeignCompanyDetails.kt
package com.babestudios.companyinfouk.data.model.company

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class ForeignCompanyDetails (

		@SerializedName("accounting_requirement")
	var accountingRequirement: AccountingRequirement? = null,

		@SerializedName("accounts")
	var accounts: ForeignCompanyAccounts? = null,

		@SerializedName("business_activity")
	var businessActivity: String? = null,

		@SerializedName("company_type")
	var companyType: String? = null,

		@SerializedName("governed_by")
	var governedBy: String? = null,

		@SerializedName("is_a_credit_finance_institution")
	var isACreditFinanceInstitution: String? = null,

		@SerializedName("originating_registry")
	var originatingRegistry: OriginatingRegistry? = null,

		@SerializedName("registration_number")
	var registrationNumber: String? = null
) : Parcelable
